jerry-core/jerry-port.h

```diff
@@ -139,6 +139,22 @@ bool jerry_port_get_time_zone (jerry_time_zone_t *tz_p);
  */
 double jerry_port_get_current_time (void);
 
+#ifndef CONFIG_DISABLE_ES2015_PROMISE_BUILTIN
+
+
+typedef struct jerry_job_t jerry_job_t;
+typedef void (*jerry_job_handler_t) (jerry_job_t *);
+
+struct jerry_job_t
+{
+  jerry_job_handler_t handler; /**< the handler funtion for the job */
+};
+
+void jerry_port_jobqueue_init (void);
+void jerry_port_jobqueue_enqueue (jerry_job_t *job_p);
+void jerry_port_jobqueue_run (void);
```

Comment:
about `jerry_port_jobqueue_init` and `jerry_port_jobqueue_run`:

You are right! Those two functions are called by the port, and jerry-core will never call them directly.